DependencyInjection/LiipMonitorExtension.php

```diff
@@ -207,18 +220,35 @@ private function configureDoctrineMigrationsCheck(ContainerBuilder $container, a
     /**
      * Return key-value array with migration version as key and class as a value defined in config file
      *
-     * @param AbstractFileConfiguration $config Current configuration
-     * @param Connection                $connection Fake connections
+     * @param ContainerBuilder               $container  The container
+     * @param DoctrineMigrationConfiguration $config     Current configuration
+     * @param Connection                     $connection Fake connections
      *
      * @return array[]
      */
-    private function getPredefinedMigrations(AbstractFileConfiguration $config, Connection $connection)
+    private function getPredefinedMigrations(ContainerBuilder $container, DoctrineMigrationConfiguration $config, Connection $connection)
     {
         $result = array();
 
-        $diff = new MigrationConfiguration($connection);
+        $diff = new LiipMigrationConfiguration($connection);
         $diff->setMigrationsNamespace($config->getMigrationsNamespace());
         $diff->setMigrationsDirectory($config->getMigrationsDirectory());
+        $diff->setContainer($container);
+        try {
+            $diff->configure();
```

Comment:
this uses container parameters under the hood, so i can't see that it can ever work, as the container is not built at this stage.

Reply:
This line only uses _kernel_ parameters and parameters, provided by doctrine bundle. The case when it will be broken right now is when user defines custom parameters for doctrine migrations bundle itself.

Reply:
does doctrine migrations bundle need to be registered first, though?

Reply:
for now yes